app/controllers/posts_controller.rb

```diff
@@ -1,12 +1,12 @@
 class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
-    @posts = @user.posts
+    @posts = @user.posts.includes(:author)
```

Comment:
- [ ] @willy836 and @RashidMenhas kindy ensure the N+1 problem is solved when fetching all posts and their comments for a user, it will mean replacing `: author` with `: comments`, since the author is a repetition of @user  and won't improve your app speed, also include screenshots of your app's logs before and after the fix in the pull request description.